pages/mydoc/notes.md

```diff
@@ -1,13 +1,38 @@
 ---
 title: "Discussion Notes"
-keywords: icedb libicedb
+keywords: icedb libicedb discussion notes
 tags: [collaboration]
 sidebar: mydoc_sidebar
 permalink: notes.html
 summary: Short summaries of the web meeting among the development team
 ---
 
-## 17 April 2018
+Since the last IPWG/IWSSM (Bologna 2016) meeting a couple of people (Stefan, Jussi, Davide, Eugene, Ben) worked on an excel-sheet with an initial file structure and variable names (following as closely as possible the CF convention) for storing scattering datasets. The Excel-file can be downloaded [here]
+
+At the recent snowfall workshop in Cologne, we discussed to better store the data in different level files:
+
+Level0: Lowest level of processing
+
+    Level0a: Contains the 3D structure
+    Level0b: Contains amplitude scattering matrix (following Mischenko convention)
+
+Level 1: Phase Matrix (to be discussed)
+
+Level 2: Essential scattering properties (to be discussed)
+
+    Details need to be further discussed
+    General idea: provide similar content as Liu database (cross sections, phase function, angle averaged quantities)
+
+Level 3: Scattering properties integrated over PSD (to be discussed)
+
+It might not be possible to generate all data levels from existing databases (some information might be missing) but it should be mandatory for all dataset developers to provide at least level0 and level2 files. For leve0a, an agreement was already found on how to name and define the 3D structure variables (all information can be found in the Excel file). Ideally, the level0 files should be enough to provide it to an API which generates the higher level files from it. Also interpolations of variables in the database requested by several users (included e.g. by Liu-DB) should be provided by an API which is separated from any specific database but can be used with any dataset.
+
+We will keep the discussions ongoing via web-meetings to work out the details of the different file types.
+
+**ENTRIES ARE IN REVERSED CRONOLOGICAL ORDER**
+
+
+## Notes from the 4th Scattering WebEx Meeting (17 April 2018)
 Webex seem to not work with pc audio but just with phones. Screen sharing is working. Webex is notoriously bad on linux platforms, it generally requires 32 bit implementations of java packages which are critically badly mantained on most distributions. It could work easier by installing a 32 bit linux distribution on a virtual machine, but still quite time consuming. 
```

Comment:
We should add a link to the meeting slides (uploaded here):
[2018-04-17 Scattering Meeting Slides.pdf](https://github.com/rhoneyager/libicedb/files/1929718/2018-04-17.Scattering.Meeting.Slides.pdf)